packages/typespec-test/test/healthinsight/generated/typespec-ts/src/outputModels.ts

```diff
@@ -18,33 +20,11 @@ export interface TrialMatcherResultOutput {
    */
   readonly status: string;
   /** An array of errors, if any errors occurred during the processing job. */
-  readonly errors?: Array<ErrorModelOutput>;
+  readonly errors?: Array<ErrorModel>;
```

Comment:
Small fix: if the error type is from core we would directly re-use the one in js core.